feature-libs/product-configurator/rulebased/styles/_configurator-add-to-cart-button.scss

```
@@ -6,15 +6,71 @@
   border-top: solid 1px var(--cx-color-light);
   box-shadow: 0 0 5px var(--cx-color-light);
 
-  .cx-add-to-cart-btn-container,
-  .cx-display-only-btn-container {
-    max-width: 1140px;
-    @include cx-configurator-footer-container();
-    margin-top: 0px;
-
-    button.cx-add-to-cart-btn,
-    button.cx-display-only-btn {
-      @include cx-configurator-footer-container-item();
+  @include forVersion(1, 6) {
```

What do those 2 parameters mean?

---

They styling changes will be only applied between SPA version 1 and 6

---

But would 
```suggestion
  @include forVersion(6) {
```
be sufficient then?

---

Done!